src/main/java/com/ernie/TicketApp/repository/UserDAO.java

```diff
@@ -93,4 +98,17 @@ public List<User> getAllUsers() {
         return allUsers;
     }
 
+    @Transactional
+    public void updateUserAddTicket(UUID id) {
+        Ticket newTicket = new Ticket();
+        newTicket.setTicketType(TicketType.MONTH);
```

Comment:
this is service layer logic. all data has to be already prepared, when you work with it on dao layer